src/playwright/pages/front.page.ts

```diff
@@ -0,0 +1,30 @@
+import type { Page } from '@playwright/test';
+
+import { BasePage } from './base.page';
+import type { Language } from '../../types';
+
+type AllowedPaths = '/' | `/${Language}`;
+
+export class FrontPage extends BasePage {
+  constructor(page: Page) {
+    super(page);
+  }
+
+  static async create(page: Page, path: AllowedPaths) {
+    const frontPage = new FrontPage(page);
+    await frontPage.page.goto(path);
+    return frontPage;
+  }
+
+  async isFinnish() {
+    await this.isTitle('Kulttuuri- ja toimintakalenteri');
```

Comment:
"isTitle" should be renamed to "hasTitle".

Reply:
Renamed in 0dc80d04f502db61b198303b14f528374b91d621